ext_conf_template.txt

```diff
@@ -0,0 +1,2 @@
+# cat=basic; type=string; label=Exclude tables from Reference Indexer via slot shouldExcludeTableFromReferenceIndex
+excludetablefromteferenceindex = fe_users, fe_groups
```

Comment:
There's a typo in this setting (`teference` instead of `reference`). Also, please re-word so it becomes `excludeTablesFromReferenceIndexing` (table**s**) and include in the label that the setting only works on 8.6+.